kraft.yaml

```
@@ -1,6 +1,6 @@
 specification: '0.5'
```

```suggestion
---
specification: '0.5'
```

Compatible with the new syntax.